core/cas-server-core-web-api/src/main/java/org/apereo/cas/web/extractcert/X509CertificateExtractorConfig.java

```diff
@@ -0,0 +1,29 @@
+package org.apereo.cas.web.extractcert;
+
+
+import lombok.extern.slf4j.Slf4j;
+import org.apereo.cas.configuration.CasConfigurationProperties;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * @since 5.3.3
+ */
+@Configuration("x509CertificateExtractorConfig")
+@EnableConfigurationProperties(CasConfigurationProperties.class)
+@Slf4j
+class X509CertificateExtractorConfig {
```

Comment:
Please correct the javadocs for the class, and make sure the class ends with `Configuration` for consistent, and that it is made public. Remove the Slf4j reference as well.